src/03-ticket-calculator.js

```diff
@@ -109,7 +210,44 @@ function calculateTicketPrice(ticketData, ticketInfo) {}
     purchaseTickets(tickets, purchases);
     //> "Ticket type 'discount' cannot be found."
  */
-function purchaseTickets(ticketData, purchases) {}
+
+  function purchaseTickets(ticketData, purchases) {
+    let totalPrice = 0
+    // Returns the general reciept without any addons.
+    let result = "Thank you for visiting the Dinosaur Museum!\n-------------------------------------------\n"
+    for (let purchase of purchases) {
+
+      // Variable that holds the price for each ticket.
+      let ticketPrice = calculateTicketPrice(ticketData, purchase)
```

Comment:
great job leveraging you calculateTicketPrice function

Reply:
Appreciate that!!!